include/swift/Basic/LangOptions.h

```diff
@@ -172,7 +172,7 @@ namespace swift {
     ///
 
     /// User-overridable language version to compile for.
-    version::Version EffectiveLanguageVersion = version::Version::getCurrentLanguageVersion();
```

Comment:
This change seems to be causing issues in CI. Out of curiosity, why don't we want the current language version here?
```
/home/build-user/swift/include/swift/Basic/LangOptions.h:175:49: error: use of undeclared identifier 'Version'
    version::Version EffectiveLanguageVersion = Version{5};
                                                ^
/home/build-user/swift/include/swift/Basic/LangOptions.h:175:56: error: expected ';' at end of declaration list
    version::Version EffectiveLanguageVersion = Version{5};
                                                       ^
```

Reply:
Not all of the swift code is Swift 6 ready yet, I have a patch to fix this issue. Also, working on getting a new PR for `swift-experimental-string-processing`

Previous error:

```
/home/build-user/swift/stdlib/public/libexec/swift-backtrace/main.swift:84:14: error: static property 'args' is not concurrency-safe because it is non-isolated global shared mutable state
  static var args = Arguments()
```